server/graphql/v2/input/AccountingCategoryInput.ts

```diff
@@ -42,6 +51,13 @@ export const AccountingCategoryInput = new GraphQLInputObjectType({
       type: GraphQLString,
       description: 'A friendly name for non-accountants (i.e. expense submitters and collective admins)',
     },
+    hostOnly: {
+      type: GraphQLBoolean,
```

Comment:
If there's a default value, we can:

```suggestion
      type: new GraphQLNonNull(GraphQLBoolean),
```